index.bs

```diff
@@ -3435,6 +3441,150 @@ The [=remote end steps=] with <var ignore>session</var> and |command parameters|
 
 </div>
 
+#### The browsingContext.print Command ####  {#command-browsingContext-print}
+
+The <dfn export for=commands>browsingContext.print</dfn> command
+creates a paginated representation of a document, and returns it as a
+Base64-encoded string.
+
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browsingContext.Print = {
+        method: "browsingContext.print",
+        params: browsingContext.PrintParameters
+      }
+
+      browsingContext.PrintParameters = {
+        context: browsingContext.BrowsingContext,
+        ?background: bool .default false,
```

Comment:
Question: shall we start using defaults for other commands as well? I haven't seen that yet and it's nice. Happy to file a follow-up issue for that.

Reply:
Yeah, if people like it then I agree it's better than prose :)

Reply:
Great. I filed https://github.com/w3c/webdriver-bidi/issues/368.